src/support/LookupSummaryAndPolicies.tsx

```diff
@@ -0,0 +1,55 @@
+import { ButtonPrimary } from '@terra-ui-packages/components';
+import React, { Fragment, useState } from 'react';
+import { TextInput } from 'src/components/input';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import { ResourcePolicies } from 'src/support/ResourcePolicies';
+import { ResourceTypeSummaryProps } from 'src/support/SupportResourceType';
+import { SupportSummary } from 'src/support/SupportSummary';
+
+export const LookupSummaryAndPolicies = (props: ResourceTypeSummaryProps) => {
+  const { query } = Nav.useRoute();
+  const [resourceId, setResourceId] = useState<string>(props.fqResourceId.resourceId);
+
+  function submit() {
+    Nav.updateSearch({ ...query, resourceId: resourceId || undefined });
+  }
+
+  return (
+    <>
+      <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'space-between', marginBottom: '1rem' }}>
+        <div
+          style={{
+            color: colors.dark(),
+            fontSize: 18,
+            fontWeight: 600,
+            display: 'flex',
+            alignItems: 'center',
+            marginLeft: '1rem',
+          }}
+        >
+          {props.displayName}
+        </div>
+        <TextInput
+          style={{ marginRight: '1rem', marginLeft: '1rem' }}
+          key={`${props.fqResourceId.resourceTypeName}-input`}
```

Comment:
A "controlled input" that has `value` set to `resourceId` might be more idiomatic than an uncontrolled input using `defaultValue` and `key` (which looks like it's there to clear the input when the resource type changes). Resetting the input could be done with an effect hook based on the resource type.

https://react.dev/reference/react-dom/components/input#controlling-an-input-with-a-state-variable